### PR TITLE
[EN/ZH] Add `Yuyuko's Hidden Cup Season 3` and Update `tournaments`

### PR DESCRIPTION
**Pull request by YumeMuzi**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

### CODE REVIEW

File: wiki/Tournaments/YHC/en.md [hunk 1]
```diff
@@ -3,3 +3,4 @@
 Index page for all of the Yuyuko's Hidden Cup series.
 
 - [Yuyuko's Hidden Cup Season 2](2)
+- [Yuyuko's Hidden Cup Season 3](3)
```
Comment (agatemosu) [inline wiki/Tournaments/YHC/en.md:6]: here is missing the first tournament (without an article)

File: wiki/Tournaments/YHC/3/en.md [hunk 2]
```diff
@@ -0,0 +1,323 @@
...
+| Elimination stage 2 | 2023-06-26/2023-07-09 |
+| Elimination stage 3 | 2023-07-10/2023-07-29 |
+
+## Prizes
+
+| Placing | Prize(s) |
+| :-: | :-- |
+| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | USD 4.8 per player, an unofficial badge |
+| ![Gold crown](/wiki/shared/crown-silver.png "2nd place") | USD 3.4 per player |
+| ![Gold crown](/wiki/shared/crown-bronze.png "3rd place") | USD 2 per player |
```
Comment (TicClick) [inline wiki/Tournaments/YHC/3/en.md:29]:
```suggestion
| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | USD 3.4 per player |
| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | USD 2 per player |
```

File: wiki/Tournaments/YHC/3/en.md [hunk 3]
```diff
@@ -0,0 +1,323 @@
...
+| *the MVP of the tournament* | 1 month of osu!supporter |
```
Comment (TicClick) [inline wiki/Tournaments/YHC/3/en.md:30]: think it's better to expand an acronym on its first occurrence for clarity
```suggestion
| *the most valuable person of the tournament (MVP)* | 1 month of osu!supporter |
```

File: wiki/Tournaments/YHC/3/en.md [hunk 4]
```diff
@@ -0,0 +1,323 @@
...
+| *the Coach of winner team* | 1 month of osu!supporter |
```
Comment (TicClick) [inline wiki/Tournaments/YHC/3/en.md:31]:
```suggestion
| *the coach of winner team* | 1 month of osu!supporter |
```

File: wiki/Tournaments/YHC/3/en.md [hunk 5]
```diff
@@ -0,0 +1,323 @@
...
+
+## Ruleset
+
+### General
+
+1. The Yuyuko's Hidden Cup Season 3 is a entertainment-oriented double elimination 4v4 team tournament, played on the osu! game mode.
+2. Beatmap scoring is based on ScoreV2.
+3. Based on the number of registration, **the size for a team is 7 players**.
+4. The beatmap list for each next round will be announced by the mappool selectors on the official live stream, on the weekend (Friday 12:00 - Monday 23:00) after the matches take place.
+5. The default match schedules for each round will be announced by the tournament managers in the information (main) sheet, on Sunday before the last matches take place. the captain could have a discussion with the opponent and confirm the match time, before Saturday 00:00 in the week.
```
Comment (TicClick) [inline wiki/Tournaments/YHC/3/en.md:213]: (if I caught your intention correctly)
```suggestion
5. The default match schedules for each round will be announced by the tournament managers in the information (main) sheet, on Sunday before the last matches take place. The captain may have a discussion with the opponent and confirm the match time before Saturday 00:00 on the same week.
```

File: wiki/Tournaments/YHC/3/en.md [hunk 6]
```diff
@@ -0,0 +1,323 @@
...
+6. Use of the Visual Settings to alter background dim or disable beatmap elements like storyboards and skins are allowed.
```
Comment (TicClick) [inline wiki/Tournaments/YHC/3/en.md:214]: alternatively, simply Visual Settings
```suggestion
6. Use of the Visual Settings panel to alter background dim or disable beatmap elements like storyboards and skins is allowed.
```

File: wiki/Tournaments/YHC/3/en.md [hunk 7]
```diff
@@ -0,0 +1,323 @@
...
+   - Custom skin elements must not be used to alter core gameplay elements or mechanics in unintended ways.
+7. If less than the minimum amount of required players are present at match time, the match can be postponed for up to 10 minutes. If after this period there are still not enough players for either team, a win by default will be declared for the side with the most members present.
+   - The minimum amount of required players is the amount of players needed to play a beatmap without any vacant spots in the lobby (i.e. 4 participants must be present for the match to begin). 
+8. Exchanging players between games is allowed without limitations.
```
Comment (TicClick) [inline wiki/Tournaments/YHC/3/en.md:218]: "exchanging" makes it sound like the teams exchange between themselves (at least to me)
```suggestion
8. Replacing players between games is allowed without limitations.
```

File: wiki/Tournaments/YHC/3/en.md [hunk 8]
```diff
@@ -0,0 +1,323 @@
...
+9. If a game ends in a draw, it will be nullified and the beatmap will be replayed.
+10. Teams may ask for a rematch if a team member encounters technical issues while playing within 30 seconds or 25% of the beatmap drain length (whichever happens first) of the game's start. Referees may, at their discretion, veto this request.
+    - "Lag spikes" are not considered a valid reason to rematch a beatmap.
+    - The roster for each team during a rematch must remain the same as the original run. If that is not possible, e.g. by virtue of a technical issue that prevents a player from entering the lobby, both teams will be allowed to swap rosters.
+    - This rule is not to be abused. Referees may veto a rematch request if they find that this is the case.
+11. If a player disconnects mid-game, their scores will not be counted towards their team's total, unless adequate proof of said score is provided. The following are considered as acceptable proof:
+    - Player point-of-view live stream snippets (commonly referred to as "clips" or "VODs"). The entirety of the play, along with the results screen must be clearly visible along with the affected player's score.
```
Comment (TicClick) [inline wiki/Tournaments/YHC/3/en.md:225]:
```suggestion
    - Player point-of-view live stream snippets (commonly referred to as "clips" or "VODs"). The entirety of the play, along with the results screen, must be clearly visible along with the affected player's score.
```

File: wiki/Tournaments/YHC/3/en.md [hunk 9]
```diff
@@ -0,0 +1,323 @@
...
+      - All screenshots **MUST** be taken using the game itself (using `Shift` + `F12`), that is, they must be hosted on the `https://osu.ppy.sh/` domain. Any other form of screenshot will be denied.
+    - Player scores may be derived from the official stream as a last resort, in cases where the match is streamed.
+12. Players are expected to keep the match running fluently and without delays. Excessive match delays from the players' side may result in penalties being applied by the tournament managers. Disrupting the match by foul play, insulting or provoking other players or staff, delaying the match, or other deliberate inappropriate misbehaviour are strictly prohibited, and will be punished accordingly.
+13. All players and staff must be treated with respect. Instructions from the referees and the tournament managers are to be followed. Decisions labelled as final are not to be objected.
+14. The multiplayer chatrooms underlie the [osu! community rules](/wiki/Rules). All chat rules apply to the multiplayer chatrooms where the matches will take place.
```
Comment (TicClick) [inline wiki/Tournaments/YHC/3/en.md:234]:
```suggestion
14. The multiplayer chatrooms abide by the [osu! community rules](/wiki/Rules). All chat rules apply to the multiplayer chatrooms where the matches will take place.
```

File: wiki/Tournaments/YHC/3/en.md [hunk 10]
```diff
@@ -0,0 +1,323 @@
...
+    - Breaking the chat rules may result in a silence. Silenced players cannot participate in multiplayer matches and must be exchanged for the duration of the punishment.
```
Comment (TicClick) [inline wiki/Tournaments/YHC/3/en.md:235]:
```suggestion
    - Breaking the chat rules may result in a silence. Silenced players cannot participate in multiplayer matches and must be replaced for the duration of the punishment.
```

File: wiki/Tournaments/YHC/3/en.md [hunk 11]
```diff
@@ -0,0 +1,323 @@
...
+   - pp limitation: 2000pp - 4500pp
+3. If a player registered successfully, a **cost** will be calculated and will affect the team-up stage.
+
+### Team-up information
+
+1. Each team will have 7 players (including the captain).
+2. The sum of player cost should be 6.7 - 7.3 (including boundary values).
+3. Please try to keep the cost within a reasonable threshold. If out of limit, staff will take action.
+4. If a team formed successfully, the captain could send the team name and team flag to the staff.
+   - The team flag and team name should comply with [osu! community rules](/wiki/Rules). sexualized, trolling, insulting, derogatory, personal, and political attack team name or team flag is not allowed.
```
Comment (TicClick) [inline wiki/Tournaments/YHC/3/en.md:259]:
```suggestion
   - The team flag and team name should comply with [osu! community rules](/wiki/Rules). Sexualized, trolling, insulting, derogatory, personally and/or politically offensive team names and flags are not allowed.
```

File: wiki/Tournaments/YHC/3/en.md [hunk 12]
```diff
@@ -0,0 +1,323 @@
...
+2. Each team may ban **one beatmap** to be selected from the pool In all stages of the tournament. These beatmaps may not be picked by any team for the entire duration of the match.
+3. Each captain must use `!roll` once in the match's chatroom.
+   - The winner of the `!roll`, the team captain would decide if their team **picks** first or **bans** first.
+   - Teams may "double pick" (i.e. pick two or more maps from the same mod pool in sequence) without limitations.
+   - The loser of the `!roll` 's ban/pick depends on the winner of the `!roll`.
+4. After bans are decided, both teams will take turns in picking a beatmap from the mappool.
+5. Teams will be allowed 2 minutes to pick a beatmap and 2 minutes to press the `Ready` button on their client. If a team takes more time than alloted for either action, the procedures adopted will be as follows:
+   - For the first occurrence:
+     - The team will receive a verbal warning from the referee
```
Comment (TicClick) [inline wiki/Tournaments/YHC/3/en.md:290]:
```suggestion
     - The team will receive a verbal warning from the referee.
```

File: wiki/Tournaments/YHC/3/en.md [hunk 13]
```diff
@@ -0,0 +1,323 @@
...
+
+1. Each stage will be held on **a single or double weekend**.
+2. All bracket stages will be held between Wednesday 00:00 and Sunday 22:00 UTC+8.
+3. Match scheduling will be handled by the tournament managers. Schedules will be released on the Sunday after the last matches of the stage. The tournament managers will try to create the schedule to respect the participants' time zones and time favors.
+   - In addition, team captains may inform tournament managers if they expect a specific time slot to be unavailable in the following week.
+4. **Reschedules will only be considered if both teams agree to a time and communicate it to the tournament managers, before Friday 23:59 UTC+8 of the week the match is to take place on.**
+   - Matches may not be scheduled to any time beyond **Sunday, 22:00 UTC+8** of the week they are to be played at.
+   - **Do not ask for a reschedule unless it is absolutely needed. The tournament managers reserve the right to deny any rescheduling request.**
+   - Late reschedule requests will not be accepted under any circumstances.
+5. Captains and Coaches are responsible for their team's availability.
```
Comment (TicClick) [inline wiki/Tournaments/YHC/3/en.md:323]:
```suggestion
5. Captains and coaches are responsible for their team's availability.
```

**Review by TicClick — CHANGES_REQUESTED** [review]
mostly minor corrections, please apply them and we're done

File: wiki/Tournaments/YHC/3/en.md (hunk 2 above)
Comment (TicClick) [reply wiki/Tournaments/YHC/3/en.md:29]: btw same issue is in the translation

Comment (YumeMuzi) [reply wiki/Tournaments/YHC/3/en.md:29]: fix it soon

**Review by TicClick — DISMISSED** [review]
should be good to go

**Comment by TicClick** [comment]
(for clarity, I withdrew my approval because I didn't see any Chinese reviewers -- the English article is fine IMO)

File: wiki/Tournaments/YHC/3/zh.md [hunk 14]
```diff
@@ -0,0 +1,325 @@
...
+  - YHC
+  - 幽幽杯
+  - 幽幽子
+---
+
+# 第三届幽幽杯
+
+![YHC S3 横幅](img/banner.jpg)
+
+**第三届幽幽杯** (***YHC S3***) 是基于 osu! 主模式，娱乐为主的多人 (4v4) 打图游玩锦标赛，主办是 ::{ flag=CN }:: [SIyuyuko](https://osu.ppy.sh/users/9794030)。这是该系列赛的第二届比赛。
```
Comment (cdwcgt) [inline wiki/Tournaments/YHC/3/zh.md:13]:
```suggestion
**第三届幽幽杯** (***YHC S3***) 是基于 osu! 主模式，娱乐为主的多人 (4v4) 打图游玩锦标赛，主办是 ::{ flag=CN }:: [SIyuyuko](https://osu.ppy.sh/users/9794030)。这是该系列赛的第三届比赛。
```

File: wiki/Tournaments/YHC/3/zh.md [hunk 15]
```diff
@@ -0,0 +1,325 @@
...
+    - 选手可以提供游戏回放文件，通过从被影响的选手的客户端的“本地”选项卡下获取到（成绩的时间戳必须与比赛链接中，该场对局的时间戳完全匹配）。
+    - 房间内的其他选手可以直接在游戏中截图，显示被影响选手的分数。
+      - 在结算界面的截图必须清晰展示被影响选手的分数。推荐使用这种方法。
+      - 被影响选手可提供断连时，在游戏中按下 F12 截下的图。注意，在游戏中截下的图上所示的分数，不能直接算入该队的总分。并且，不推荐使用这种方法。如果这张截图不足以证明是该名选手断连时的截图，则裁判可自行决定。
+      - 所有截图**必须**使用 `Shift` + `F12` 上传到 osu! 官网。必须保证网络截图的域名是 `https://osu.ppy.sh/` 。其他域名的截图无效。
+    - 如果比赛正在直播，那么作为最后的手段，可以从直播的观赛端读取选手最后留下的分数。
+12. 选手应协助比赛顺畅进行，尽量少因为网络问题而延误。如果某一队的延误情况严重，则可能受到裁判或主办的惩罚。严禁犯规、侮辱或者挑衅其他选手或者裁判、主办、工作人员等，严禁故意拖延比赛或扰乱比赛。相应者会受到处罚。
+13. 应该尊重所有选手或者裁判、主办、工作人员等人。必须遵守裁判和主办的意见，不得反驳他们提出的最终决定。
+14. 比赛将在 osu! 官方服务器进行。因此，在多人游戏房间中，必须遵守 [osu! 社区守则](/wiki/Rules)。
+    - 如果有刷屏等等的行为，选手将会被禁言。他们无法进入比赛房间。只能派上其他选手替代他的位置。
```
Comment (cdwcgt) [inline wiki/Tournaments/YHC/3/zh.md:237]:
```suggestion
    - 如果有刷屏等等违反社区规定的行为，选手将会被禁言。他们无法进入比赛房间。只能派上其他选手替代他的位置。
```

https://osu.ppy.sh/wiki/en/Silence
不知道有没有必要加上链接，总之挂在这里

File: wiki/Tournaments/YHC/3/zh.md [hunk 16]
```diff
@@ -0,0 +1,325 @@
...
+15. 违反比赛规则的处罚包括但不限于：
+    - 禁止某位选手上一张图。
```
Comment (cdwcgt) [inline wiki/Tournaments/YHC/3/zh.md:239]:
```suggestion
    - 禁止某位选手游玩某一张图。
```

**Review by cdwcgt — CHANGES_REQUESTED** [review]
just checked text, should be no problem except the following content.

Consider adding a note that the article timeline defaults to UTC+8?

File: wiki/Tournaments/YHC/3/zh.md (hunk 15 above)
Comment (YumeMuzi) [reply wiki/Tournaments/YHC/3/zh.md:237]: 无所谓 no matter

**Comment by YumeMuzi** [comment]
> (for clarity, I withdrew my approval because I didn't see any Chinese reviewers -- the English article is fine IMO)

Now we should be able to merge, and thank you for checking the English article

**Review by TicClick — APPROVED** [review]

**Pull request merged**